src/node_i18n.cc

```diff
@@ -519,14 +519,22 @@ static void GetStringWidth(const FunctionCallbackInfo<Value>& args) {
   }
 
   TwoByteValue value(env->isolate(), args[0]);
+  if (value.length() <= 1) {
```

Comment:
This additional shortcut is wrong. See the CI failures (https://ci.nodejs.org/job/node-test-commit-osx/5807/nodes=osx1010/console).